salt/grains/core.py

```diff
@@ -2760,26 +2768,6 @@ def _hw_data(osdata):
         else:
             log.error('The \'prtconf\' binary was not found in $PATH.')
 
-    elif osdata['kernel'] == 'AIX':
```

Comment:
Is it really okay to remove this part for AIX? Wouldn't they need it anymore?

Reply:
Yes, check a bit early in line: 2743. This code comes from broken rebase for those merge-forward PR